content/graph/WeightedMatching.h

```diff
@@ -1,89 +1,44 @@
 /**
- * Author: Stanford
- * Date: Unknown
- * Source: Stanford Notebook
- * Description: Min cost bipartite matching. Negate costs for max cost.
- * Time: O(N^3)
- * Status: tested during ICPC 2015
+ * Author: Benjamin Qi, Chilli
+ * Date: 4/04/20
+ * License: CC0
+ * Source: https://github.com/bqi343/USACO/blob/master/Implementations/content/graphs%20(12)/Matching/Hungarian.h
+ * Description: Minimum cost bipartite matching. Takes in cost[N][M], where N
+ * <= M, and cost[i][j] = cost for i-th node on left to j-th node on right.
```

Comment:
say something about every node on the left choosing one on the right such that no two choose the same node. I.e., make this intelligible without knowing what it does